zadaci/sa-kolokvijuma/2020/RA/T34/G4/zadatak.rst

```diff
@@ -0,0 +1,63 @@
+Iz zadate ulazne datoteke ``posiljke.txt`` učitati statički niz
+struktura, pri čemu se struktura ``posiljka_st`` sastoji od sledećih
+polja:
+
+-  ``sifra`` (jedna reč, maksimalno 5 karaktera),
+-  ``zemlja`` (jedna reč, maksimalno 10 karaktera),
+-  ``vrednost`` (jedna reč, maksimalno 4 karaktera) – izražena u evrima,
+-  ``tezina`` (realan broj) – izražena u gramima i
+-  ``cena`` (realan broj).
+
+Na osnovu parametra ``odabrana_zemlja`` koji se prihvata preko standardnog
+ulaza, izračunati cenu pošiljke nakon carine na sledeći način:
+
+-  ako je vrednost pošiljke veća od 100 evra, težina veća od 1000 grama
```

Comment:
Please fix your RST formatting as it's not being rendered correctly (unnecessary bold text), see the attached screenshot
![Screenshot from 2021-04-10 10-12-35](https://user-images.githubusercontent.com/371851/114263387-6e2dfc00-99e5-11eb-8461-5f1a2bdddfea.png)

Reply:
Updated 
![images](https://user-images.githubusercontent.com/44647058/114276335-7a867900-9a26-11eb-8c69-338db10290ed.JPG)